TodoList/Tests/TodoList.Api.Tests/TodoList.Api.Tests.csproj

```diff
@@ -59,9 +74,28 @@
     </Reference>
     <Reference Include="System.Web.Routing" />
     <Reference Include="System.Xml" />
+    <Reference Include="Unity.Abstractions, Version=3.3.1.0, Culture=neutral, PublicKeyToken=6d32ff45e0ccc69f, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Unity.5.8.11\lib\net47\Unity.Abstractions.dll</HintPath>
+    </Reference>
+    <Reference Include="Unity.Configuration, Version=5.2.5.0, Culture=neutral, PublicKeyToken=6d32ff45e0ccc69f, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Unity.5.8.11\lib\net47\Unity.Configuration.dll</HintPath>
+    </Reference>
+    <Reference Include="Unity.Container, Version=5.8.11.0, Culture=neutral, PublicKeyToken=489b6accfaf20ef0, processorArchitecture=MSIL">
```

Comment:
It's really cool to comment on a code that is not related to what I want to tell you 👍

No, it isn't so cool 😡.

Reply:
Next time it will be better to add a link to `test cases`. I met that two weeks ago on Coding Dojo where I have learned that it exists. But without that, I will not know what you meant.

5e496f2

Reply:
You'd ask ☺️ (I'll try to think about that.)

Reply:
Well, you could actually google it first and then ask 😇 😆

Reply:
But for the reviewer (who knows what he wants to advice) it is 10 second for adding the link. But for a reader, it could be a few minutes of grope 😕.

Reply:
Learning is not about a destination, but (mostly) about the journey 😛 In other words (that I just copied): [授人以魚不如授人以漁](https://en.wiktionary.org/wiki/give_a_man_a_fish_and_you_feed_him_for_a_day;_teach_a_man_to_fish_and_you_feed_him_for_a_lifetime) 😆 

On more serious note: it depends - while looking up links and providing best possible articles is useful to people not knowing what you are speaking about, espeacially topics that are often aproach from many different directions. Providing it to a person that knows might be perceived as offending or bothering and waste of time for both sides; especially if the topic is deemed "basic". I would personally take it as a compliment that link wasn't there, but that's just me 😊